spring-batch/src/main/java/org/baeldung/conditionalflow/NumberInfoDecider.java

```diff
@@ -0,0 +1,19 @@
+package org.baeldung.conditionalflow;
+
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.job.flow.FlowExecutionStatus;
+import org.springframework.batch.core.job.flow.JobExecutionDecider;
+
+public class NumberInfoDecider implements JobExecutionDecider {
```

Comment:
The formatting is off on this class. Be sure to use for example our eclipse formatted:
- https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml
I believe there is one similar for IntelliJ as well.

Reply:
Done